### PR TITLE
feat: add stats variable to panel title

### LINKED ISSUE #35870: Panel statistic information or Total Count for Table View
**What would you like to be added**:
In general, to be able show some panel statistic information on dashboard.

To be specific, i want to display total count of rows in table panel. 
something like
image

p.s. found some workaround: write variables in panel name, so for that i need a variable with value of total records in table
image

**Why is this needed**:
To know some info statistic info about panel (total count rows)

### PR DESCRIPTION
**Pull request by myml**
**What this PR does / why we need it**:

Pass the stats variable to getDisplayTitle function.

Used to display query stats info in the panle title.

image

**Which issue(s) this PR fixes**:

Fixes #35870

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx [hunk 1]
```diff
@@ -28,7 +28,18 @@ export interface Props {
 
 export const PanelHeader: FC<Props> = ({ panel, error, isViewing, isEditing, data, alertState, dashboard }) => {
   const onCancelQuery = () => panel.getQueryRunner().cancelQuery();
-  const title = panel.getDisplayTitle();
+  const statsExtendVar = {
+    queries_number: data.series.length,
```
Comment (torkelo) [inline public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx:32]: We do not use snake case in Grafana

Comment (torkelo) [inline public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx:32]: This is not true , the series length has nothing to do with number of queries. A single query can result in 0-N series

**Review by torkelo — COMMENTED** [review]
This is interesting, is there a feature to gauge interest in this feature?

**Review by axelavargas — COMMENTED** [review]
Hi @myml 👋🏾, I shared this PR with the team as well, and after some discussion, we are not sure about adding these additional stat variables in the panel title.

We think that the use case of the problem you are trying to solve is missing. We were guessing some use cases like "I want to know that this query takes 5 minutes to run before I hit "refresh", without the need to check the query inspector", but of course, this is an assumption.

I noticed there is a [GitHub discussion](https://github.com/grafana/grafana/discussions/38304) attached to this issue, could you please add the problems you are trying to solve in detail?

With the updated information, we will assess if the community is interested in this feature and we can think about how we could implement it (this may also require some input from our UX team.)

For now I will close the PR, if something changes on our side we can always reopen it.

Thank you very much for contributing, these PRs always open good discussions on how we can improve Grafana for the community 🙏🏾

**Comment by myml** [comment]
@axelavargas  I want to display the query total in the table panle header, like https://github.com/grafana/grafana/issues/35870.
What do you think?

**Pull request closed without merging**